js/utils/synthutils.js

```diff
@@ -11,7 +11,7 @@
 
 const POLYCOUNT = 3;
 
-var NOISENAMES = [
+let NOISENAMES = [
```

Comment:
I think it would be nice to have all these objects as `const` because at no point in the code they will undergo any alteration.

Reply:
Yes. I did a `git grep` and they aren't reassigned.